tests/unit/nupic/research/sp_learn_inference_test.py

```diff
@@ -108,7 +109,8 @@ def _runLearnInference(self,
       if spVerbosity > 0:
         print "Input #%d" % i
       encodedInput = inputs[i]
-      spLearnInfer.compute(encodedInput, learn=True, infer=False)
+      decodedOutput = np.zeros(columnDimensions)
+      spLearnInfer.compute(encodedInput, learn=True, activeArray=decodedOutput)
```

Comment:
Please file an issue for allowing `activeArray` to be `None` to allow no-inference calls and add a comment here (and anywhere else you had to do this) with a link to the issue.